tests/pipelines/test_pipelines_text_to_audio.py

```diff
@@ -66,6 +66,30 @@ def test_small_musicgen_pt(self):
         audio = [output["audio"] for output in outputs]
         self.assertEqual([ANY(np.ndarray), ANY(np.ndarray)], audio)
 
+    @slow
```

Comment:
thanks for adding a test!

Reply:
Thanks for the test, this is a really small nit, but could you rename `music_generator` to a name more relevant ?